cmake/Modules/CocosBuildHelpers.cmake

```diff
@@ -231,26 +241,18 @@ macro(cocos_pak_xcode cocos_target)
     set(MACOSX_BUNDLE_LONG_VERSION_STRING ${COCOS_APP_LONG_VERSION_STRING})
     set(MACOSX_BUNDLE_SHORT_VERSION_STRING ${COCOS_APP_SHORT_VERSION_STRING})
 
-    message("cocos package: ${cocos_target}, plist file: ${COCOS_APP_INFO_PLIST}")
+    message(STATUS "cocos package: ${cocos_target}, plist file: ${COCOS_APP_INFO_PLIST}")
 
    cocos_config_app_xcode_property(${cocos_target})
 endmacro()
 
 # set Xcode property for application, include all depend target
 macro(cocos_config_app_xcode_property cocos_app)
-    cocos_config_target_xcode_property(${cocos_app})
-    # for example, cocos_target: cpp-tests link engine_lib: cocos2d
-    get_target_property(engine_libs ${cocos_app} LINK_LIBRARIES)
-    foreach(engine_lib ${engine_libs})
-        if(TARGET ${engine_lib})
-            cocos_config_target_xcode_property(${engine_lib})
-            # for example, engine_lib: cocos2d link external_lib: flatbuffers
-            get_target_property(external_libs ${engine_lib} LINK_LIBRARIES)
-            foreach(external_lib ${external_libs})
-                if(TARGET ${external_lib})
-                    cocos_config_target_xcode_property(${external_lib})
-                endif()
-            endforeach()
+    set(depend_libs)
+    search_depend_libs_recursive(${cocos_app} depend_libs)
+    foreach(depend_lib ${depend_libs})
```

Comment:
Do we have any dll for ios/xcode? 
I thought that all libraries are static (like for Android)